source/server/getting-started/configuration/sponge-conf.rst

```diff
@@ -13,464 +13,1165 @@ Below is a list of available config settings and their comments inside the globa
-    Additional directory to search for plugins, relative to the
-    execution root or specified as an absolute path.
-    Note that the default: "${CANONICAL_MODS_DIR}/plugins"
-    is going to search for a plugins folder in the mods directory.
-    If you wish for the plugins folder to reside in the root game
-    directory, change the value to "${CANONICAL_GAME_DIR}/plugins".
-
-logging
-=======
-
-block-break
-    Log when blocks are broken
-block-modify
-    Log when blocks are modified
-block-place
-    Log when blocks are placed
-block-populate
-    Log when blocks are populated in a chunk
-block-tracking
-    Log when blocks are placed by players and tracked
-chunk-gc-queue-unload
-    Log when chunks are queued to be unloaded by the chunk garbage collector.
-chunk-load
-    Log when chunks are loaded
-chunk-unload
-    Log when chunks are unloaded
-entity-collision-checks
-    Whether to log entity collision/count checks
-entity-death
-    Log when living entities are destroyed
-entity-despawn
-    Log when living entities are despawned
-entity-spawn
-    Log when living entities are spawned
-entity-speed-removal
-    Whether to log entity removals due to speed
-exploit-itemstack-name-overflow
-    Log when server receives exploited packet with itemstack name exceeding string limit.
-exploit-respawn-invisibility
-    Log when player attempts to respawn invisible to surrounding players.
-exploit-sign-command-updates
-    Log when server receives exploited packet to update a sign containing commands from player with no permission.
-log-stacktraces
-    Add stack traces to dev logging
-world-auto-save
-    Log when a world auto-saves its chunk data. Note: This may be spammy depending on the auto-save-interval configured for world.
-
-metrics-collection
+**broken-network-handler-mods**
+
+| A list of mod ids that have broken network handlers (they interact with the game from a Netty handler thread).
+| All network handlers from a forcibly scheduled to run on the main thread.
+| Note that this setting should be considered a last resort, and should only be used as a stopgap measure while waiting for a mod to properly fix the issue.
+
+* **Type:** ``List<String>``
+
+BungeeCordCategory
 ==================
 
-default-permission
-    Determines whether newly added plugins can collect server metrics by default
-plugin-permissions
-    Per-plugin toggles indicating whether a plugin is allowed to collect server metrics
+**ip-forwarding**
 
-modules
-=======
+| If 'true', allows BungeeCord to forward IP address, UUID, and Game Profile to this server.
 
-movement-checks
-    Allows configuring Vanilla movement and speed checks
-realtime
-    Use real (wall) time instead of ticks as much as possible
-tileentity-activation
-    Controls block range and tick rate of tileentities.
-    Use with caution as this can break intended functionality.
+* **Type:** ``boolean``
+* **Default:** ``false``
 
-movement-checks
-===============
+PhaseTrackerCategory
```

Comment:
The section in the original `global.conf` file (below) is still called "cause-tracker"

Reply:
This title refers to the class which backs the `cause-tracker` entry. I added a note for clarification to the top.